src/status_im/ui/screens/network_settings/events.cljs

```diff
@@ -28,6 +29,6 @@
 
 (handlers/register-handler-fx
   :connect-network
-  (fn [_ [_ network]]
-    {:dispatch-n [[:account-update {:network network}]
-                  [:navigate-to-clean :accounts]]}))
\ No newline at end of file
+  (fn [cofx [_ network]]
+    (merge (accounts-events/account-update cofx {:network network})
+           {:dispatch-n [[:navigate-to-clean :accounts]]})))
```

Comment:
No need to `:dispatch-n` when only one event is being dispatched, `:dispatch` is sufficient

Reply:
done